Rudder-Appsflyer.podspec

```diff
@@ -12,23 +15,32 @@ Rudder is a platform for collecting, storing and routing customer event data to
                        DESC
 
   s.homepage         = 'https://github.com/rudderlabs/rudder-integration-appsflyer-ios'
-  s.license          = { :type => "Apache", :file => "LICENSE" }
-  s.author           = { 'RudderStack' => 'arnab@rudderlabs.com' }
+  s.license          = { :type => "Apache", :file => "LICENSE.md" }
+  s.author           = { 'RudderStack' => 'sdk@rudderstack.com' }
```

Comment:
Do we have this email? I believe we have `sdk-accounts@rudderstack.com`.

Reply:
yes, we do have this